### PR TITLE
JENKINS-37612# Cleanup POM and prepare for beta

### PR DESCRIPTION
**Pull request by vivek**
# Description

See https://issues.jenkins-ci.org/browse/JENKINS-37612

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests

NA - only pom version and description changed

- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
NA
- [x] Ran Acceptance Test Harness against PR changes.

Triggered https://ci.blueocean.io/job/Acceptance%20Tests%20Param/52/

# Reviewer checklist
- [x] Run the changes and verified the change matches the issue description
- [x] Reviewed the code
- [x] Verified that the appropriate tests have been written or valid explaination given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by kzantow** [comment]
LGTM :bee:

**Comment by tfennelly** [comment]
🐝 

Hurray ... we eventually renamed `blueocean-plugin` to `blueocean` !!!!

File: blueocean-commons/pom.xml [hunk 1]
```diff
@@ -5,14 +5,14 @@
   <parent>
     <groupId>io.jenkins.blueocean</groupId>
     <artifactId>blueocean-parent</artifactId>
-    <version>1.0-alpha-9-SNAPSHOT</version>
+    <version>1.0.0-b01-SNAPSHOT</version>
```
Comment (i386) [inline blueocean-commons/pom.xml]: @vivek why `-b01` instead of `-b1`?

**Comment by michaelneale** [comment]
This isn't quite what was described in the ticket. 

The ATH won't run as the module name is blueocean-plugin:

```
+ ./run.sh -a=../blueocean/blueocean-plugin/
/home/ubuntu/slave/workspace/Acceptance Tests Param/blueocean/blueocean-plugin /home/ubuntu/slave/workspace/Acceptance Tests Param/acceptance
warning: failed to load external entity "pom.xml"
warning: failed to load external entity "pom.xml"
/home/ubuntu/slave/workspace/Acceptance Tests Param/acceptance
```

If this is to be refactored, there are a few places that will need to change. 
Also whilst the module entry was renamed, the dependencies in the parent pom.xml are not right so it still is a :ant: 



BTW it was always

**Comment by tfennelly** [comment]
@michaelneale ah good catch on the missing bits.

One of the most irritating thing about `blueocean-plugin` was the name of that folder i.e. after a checkout of the repo you ended up having "blueocean-plugin/blueocean-plugin" ... yeuck. This fixes that, so yipee 😄

**Comment by michaelneale** [comment]
@tfennelly true - but need to find all the places that refer to it (readmes, ath scripts and anything else?)

Yeah it is confusing (there is a ticket open on hpi plugin that will mean you can run it from root, so I that is why I am less fussed about changing the name).

**Comment by vivek** [comment]
References to bluocean-plugin module fixed.

**Comment by michaelneale** [comment]
@vivek this will need to be updated after the last alpha of course.

**Comment by vivek** [comment]
@michaelneale yes

**Comment by michaelneale** [comment]
One this is updated to master, I think we can :bee: this - but if this is to be merged, please note the following will need to be changed: 

https://ci.blueocean.io/job/Acceptance%20Test%20PR/
https://ci.blueocean.io/job/Acceptance%20Tests%20Param/
https://ci.blueocean.io/job/Acceptance-Tests/

```
sh 'cd acceptance && ./run.sh -a=../blueocean/blueocean-plugin/'
```
should become: 
```
sh 'cd acceptance && ./run.sh -a=../blueocean/blueocean/'
```

**Comment by vivek** [comment]
Merged latest master, ran ATH locally, all good. @michaelneale If we are releasing tomorrow we should get it finalized?

**Comment by i386** [comment]
@vivek so we have to do `-b01`?

**Comment by vivek** [comment]
@i386 to be on safer side yes, soon we will be in double digit and will be a non issue.

**Comment by michaelneale** [comment]
@vivek +1 just sent email giving the OK, whenever it suits.

**Pull request merged**